_data/site.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "USWDS Sandbox",
+  "baseurl": "",
```

Comment:
`baseurl` is from previous data. Need to consider removing to avoid confusion in eleventy.